### ISSUE TITLE
In recent alpha snapshots when any NVDA dialog is open configuration profiles triggers are disabled.

### ISSUE THREAD

**Issue by lukaszgo1** [nvaccess/nvda#8821]
### Steps to reproduce:
1. Create a configuration profile and trigger which will activate it when switching to a given app.
2. Make a noticeable change in this profile and save it.
3. Open any NVDA dialog in this example speech with CTRL+NVDA+v.
4. Switch to the app with  a configuration profile 
### Actual behavior:
The profile is not activated.
### Expected behavior:
The profile should be activated.
#### NVDA Installed/portable/running from source:
Installed
#### NVDA version:
alpha-16149,e713f614
#### Windows version:
Windows 7 x64
#### Name and version of other software in use when reproducing the issue:
Becky Internet Maill as an app with a triggered profile.
#### Does the issue still occur after restarting your PC?
Yes
#### Have you tried any other versions of NVDA?

No.

**Comment by leonardder** [comment]
I can not reproduce this with alpha-16149 when using a profile for Notepad.

**Comment by lukaszgo1** [comment]
It wasn't as simple to reproduce as i thought. I've updated an initial commend accordingly.

**Comment by leonardder** [comment]
Thanks for your initial comment. However, this is not a bug.

1. gui.settingsDialogs.SettingsDialog has a property shouldSuspendConfigProfileTriggers, which defaults to True.
2. When this property is True, opening an NVDA settings dialog will automatically disable config profiles as long as the dialog is open. If this wouldn't happen, you might accidentally end up changing the wrong config profile. Imagine the case where you open the NVDA settings from the cmd profile and switching to Notepad activates the Notepad profile. What is switching back to the settings dialog supposed to do?

Now, it might make sense to, instead of having a shouldSuspendConfigProfileTriggers attribute on the SettingsDialog class, have the settings dialog remember from which application it was opened. However, that will probably add much complexity without improving the user experience in a major way. Therefore, I'm tempted to close this.

**Comment by leonardder** [comment]
you're right. There's certainly something that ought to be addressed here.

**Comment by Adriani90** [comment]
I think this is a duplicate of #10476.
